src/shared/components/chevron/Chevron.tsx

```diff
@@ -26,6 +26,7 @@ export type Direction = 'up' | 'down' | 'left' | 'right';
 export type Props = {
   direction: Direction;
   animate: boolean;
+  isDisabled?: boolean;
```

Comment:
I am very conflicted about the concept of a Chevron, which has so far been just a smart icon, which knew how to rotate, also learning about how to be disabled. If only we had a dedicated component that was a ChevronButton...